<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorView</class>
 <widget class="QWidget" name="DoctorView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>医生管理</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget {
        background-color: #f9f9f9;
        font-family: &quot;Arial&quot;, sans-serif;
        color: #333;
    }
    QLineEdit {
        border: 1px solid #ccc;
        border-radius: 5px;
        padding: 5px;
    }
    QPushButton {
        background-color: #007bff;
        color: white;
        border: none;
        border-radius: 5px;
        padding: 8px 15px;
    }
	QComboBox {
        background-color: #007bff;
        color: white;
        border: none;
        border-radius: 5px;
        padding: 8px 15px;
    }
    QPushButton:hover {
        background-color: #0056b3;
    }
    QTableView {
        border: 1px solid #ddd;
        background-color: white;
        gridline-color: #eee;
        selection-background-color: #cce5ff;
        selection-color: #000;
    }
    QLabel {
        font-size: 14px;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutQuery">
     <item>
      <widget class="QLineEdit" name="textSearch">
       <property name="placeholderText">
        <string>输入名字查询</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btSearch">
       <property name="text">
        <string>查找</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayoutButtons">
     <item row="0" column="0">
      <widget class="QPushButton" name="btAdd">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btDelete">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btEdit">
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btImport">
       <property name="text">
        <string>导入</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btExport">
       <property name="text">
        <string>导出</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>按年龄顺序</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按年龄逆序</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按职称</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutPagination">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btPrev">
       <property name="text">
        <string>上一页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="totalAndCurrent">
       <property name="text">
        <string>第X页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btNext">
       <property name="text">
        <string>下一页</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
